plutus-ledger-api/src/PlutusLedgerApi/Common/ProtocolVersions.hs

```diff
@@ -60,10 +61,14 @@ vasilPV = ProtocolVersion 7 0
 valentinePV :: ProtocolVersion
 valentinePV = ProtocolVersion 8 0
 
+-- | Conway era was introduced in protocol version 9.0
+conwayPV :: ProtocolVersion
+conwayPV = ProtocolVersion 9 0
+
 -- | The set of protocol versions that are "known", i.e. that have been released
 -- and have actual differences associated with them.
 knownPVs :: Set.Set ProtocolVersion
-knownPVs = Set.fromList [ shelleyPV, allegraPV, maryPV, alonzoPV, vasilPV, valentinePV ]
+knownPVs = Set.fromList [ shelleyPV, allegraPV, maryPV, alonzoPV, vasilPV, valentinePV, conwayPV ]
```

Comment:
@michaelpj - apologies in advance for making a comment about variable names. i'm probably one of the worst namers of variables. in general, there is a lot of confusion about cardano naming. i'm culpable for this. i [tried](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0059) to right/write my wrongs.

do you want these to name releases if named, and eras if not named? if so, then `conwayPV` should be renamed `changPV`. conway, like babbage, is an era name. while chang, like vasil, will-be/is a release name.

Reply:
I think that both a) we are confused about this, and b) it makes some sense for us to link them to ledger eras, since our stuff typically appears in a new ledger era.

It seems like there is no correct naming that works for protocol versions apart from the numbers, though? I can't find a listing of "release names" anywhere you pointed me?

Reply:
if you want eras, go with `conwayPV`. also also do not know of a good listing of "release names" besides [this](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0059/feature-table.md).